### PR TITLE
Set up Ubuntu 20 LTS instance

### PR DESCRIPTION
**Pull request by gidsg**
https://trello.com/c/cacoWXHC/63-5-upgrade-the-version-of-python-on-jenkins-to-at-least-38

This sets up a new Jenkins instance which is running Ubuntu 20 LTS which includes Python 3.8 in the package manager.

### CODE REVIEW

File: terraform/accounts/main/jenkins.tf [hunk 1]
```diff
@@ -22,6 +22,20 @@ module "jenkins_2" {
   log_bucket_name                   = module.jenkins_elb_log_bucket.bucket_id
 }
 
+module "jenkins_3" {
+  source                            = "../../modules/jenkins/jenkins"
+  name                              = "jenkins3"
+  aws_account_and_jenkins_login_ips = var.aws_account_and_jenkins_login_ips
+  jenkins_public_key_name           = aws_key_pair.jenkins.key_name # Or ${var.ssh_key_name}
```
Comment (bjgill) [inline terraform/accounts/main/jenkins.tf:29]: `# Or ${var.ssh_key_name}` - what does this mean?

File: terraform/modules/jenkins/jenkins/ec2.tf [hunk 2]
```diff
@@ -9,10 +9,14 @@ resource "aws_instance" "jenkins" {
   iam_instance_profile    = var.jenkins_instance_profile
   key_name                = var.jenkins_public_key_name
   vpc_security_group_ids  = [aws_security_group.jenkins_instance_security_group.id]
+
```
Comment (bjgill) [inline terraform/modules/jenkins/jenkins/ec2.tf:12]: Why this new blank line?

**Review by bjgill — APPROVED** [review]
Good catch for the root volume.

Just a couple of minor things - happy for you to merge once addressed.

File: terraform/accounts/main/jenkins.tf (hunk 1 above)
Comment (gidsg) [reply terraform/accounts/main/jenkins.tf:29]: @bjgill this is a copy-paste from the current instance block, but appears to be a reference that you may wish to use your own variable (as defined in https://github.com/alphagov/digitalmarketplace-aws/blob/889d2e369bbd25c939f69d8bec39aa6c4143938d/terraform/accounts/main/jenkins.tf#L2-L3)

File: terraform/modules/jenkins/jenkins/ec2.tf (hunk 2 above)
Comment (gidsg) [reply terraform/modules/jenkins/jenkins/ec2.tf]: removed in 889d2e3.

**Review by bjgill — APPROVED** [review]

**Pull request merged**